bin/compile

```diff
@@ -35,16 +38,38 @@ cp -n $BP_DIR/scripts/Procfile $BUILD_DIR/Procfile
 echo "-----> Copying static.json to application slug"
 cp -n $BP_DIR/scripts/static.json $BUILD_DIR/static.json
 
-echo "-----> Installing Bower package manager"
 cd $BUILD_DIR
-npm install bower
+if [ -d "./bower_components" ]; then
```

Comment:
I have opinions! Feel free to disregard.

I'm not sure it's a big optimization though? It kind of sucks to divert off the default path. `ember-cli` has never generated a `package.json` with `bower` in it, so asking a user to do it is adding additional steps for them to use this buildpack.